packages/stargate/src/accounts.ts

```diff
@@ -36,8 +36,13 @@ function accountFromBaseAccount(input: BaseAccount): Account {
 }
 
 /**
- * Takes an `Any` encoded account from the chain and extracts some common
- * `Account` information from it. This is supposed to support the most relevant
+ * Represents a generic function that takes an `Any` encoded account from the chain
+ * and extracts some common `Account` information from it.
+ */
+export type AccountParser = (any: Any) => Account;
```

Comment:
Ah, we probably want to make that public as it is part of a public interface, so it needs to be added to `index.ts` of the package.

Reply:
Done :+1: